pkg/quantile/ddsketch_test.go

```diff
@@ -7,19 +7,20 @@ package quantile
 
 import (
 	"fmt"
-	"math"
 	"testing"
 
 	"github.com/DataDog/sketches-go/ddsketch"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/opentelemetry-mapping-go/pkg/internal/sketchtest"
 )
 
 const (
-	acceptableFloatError = 2e-12
+	acceptableFloatError = 2e-11
```

Comment:
Had to bump this after adding exponential and normal tests. I think it's just a fact that the `math.Exp` and `math.Erf`/`math.InvErf` functions do not have a lot of accuracy